util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerDbManager.java

```diff
@@ -804,10 +802,6 @@ public String insertWgetContentHash(DataSource ds, String sviString) {
           throw new RuntimeException(e);
         }
 
-        // calculate SHA384 hash of the file content.
```

Comment:
This line would be useful if we re-enable the WGET support.

Reply:
Updated.